### ISSUE TITLE
Keep insert paramters in url

### ISSUE THREAD

**Issue by marwin1991** [libamtrack/web#169]
Now you can't share your configuration of function, we could store inserted paramets in url to let user send url to someone. Probably it will not be esay to do it. What do you think about it?

**Comment by grzanka** [comment]
Something like wolfram alfa has ?

https://www.wolframalpha.com/input/?i=3.15+*+10%5E-4+mg+%2F+cm%5E3+*+pi+*+(12.7+mm+%2F+2)%5E3

**Comment by marwin1991** [comment]
Yes, but we have to keep informations like someone inserted some values submitted, then changed it and again submitted. In the beginning we can just keep values that was inserted and then exapnd it to keep historical plots.
